pyramid_oereb/lib/records/plr.py

```diff
@@ -27,13 +27,13 @@ class PlrRecord(EmptyPlrRecord):
     Public law restriction record.
     """
 
-    def __init__(self, theme, information, law_status, published_from, responsible_office, symbol,
```

Comment:
type_code_list should be dictionary type for multilingual usage now

Reply:
Thank for your review - it was not clear to me but I will do this change now